### PR TITLE
Hyundai: add missing FW versions for Kona EV 2022

### PR DESCRIPTION
**Pull request by cbgta**
Hyundai Kona EV 2022 EU

### CODE REVIEW

**Comment by jnewb1** [comment]
have a route?

**Comment by cbgj** [comment]
@jnewb1 d4bf28312ec6d0a3|2023-10-06--17-07-16--0

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -1496,8 +1496,10 @@ def match_fw_to_car_fuzzy(live_fw_versions) -> Set[str]:
       b'\xf1\x8758520-K4010\xf1\x00OS IEB \x02 101 \x11\x13 58520-K4010',
       b'\xf1\x8758520-K4010\xf1\x00OS IEB \x04 101 \x11\x13 58520-K4010',
       b'\xf1\x8758520-K4010\xf1\x00OS IEB \x03 101 \x11\x13 58520-K4010',
+      b'\xf1\x8758520-K4010\xf1\x00OS IEB \x03 102"\x05\x16 58520-K4010',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:1499]:
```suggestion
```

Comment (cbgj) [reply selfdrive/car/hyundai/values.py:1499]: This part is different? \x03 102"\x05\x16

**Pull request merged**